Automattic/newspack-plugin
# feat: remove Updates wizard; replace with a link to manual release notes

### All Submissions:






### Changes proposed in this Pull Request:



The changelog-parsing Updates wizard was not a great idea – the language of commit messages is too technical. On top of that, we've started posting manual release notes, which are a much better way of communicating changes to users. This PR removes the Updates wizard and updates the footer link to point to the manual updates page.

### How to test the changes in this Pull Request:

1. Click the "Newspack <version>" link in the footer 
2. Observe the [manual updates page](https://newspack.pub/category/release-notes/) opening in a new tab 

### Other information:
